jenkins.go

```diff
@@ -210,6 +210,32 @@ func (j *Jenkins) CreateNode(name string, numExecutors int, description string,
 	return nil, errors.New(strconv.Itoa(resp.StatusCode))
 }
 
+// Create a new folder
+// This folder can be nested in other parent folders
+// Example: jenkins.CreateFolder("newFolder", "grandparentFolder", "parentFolder")
+func (j *Jenkins) CreateFolder(name string, parents ...string) (*Folder, error) {
+	folderObj := &Folder{Jenkins: j, Raw: new(FolderResponse), Base: "/job/" + strings.Join(append(parents, name), "/job/")}
```

Comment:
How does the end URL look like if for example I call the function like this: jenkins.CreateFolder("newFolder", "grandparentFolder", "parentFolder"). Should it look like "https://jenkins_url/job/grandparentFolder/job/parentFolder/createItem"?